src/components/Markdown/MarkdownEditor/MarkdownEditor.tsx

```diff
@@ -50,22 +44,23 @@ export const MarkdownEditor: FC<MarkdownEditorProps> = ({
   className
 }) => {
   return (
-    <div
-      className={cx(styles.Container, className, {
-        Secondary: variant === MarkdownEditorVariants.SECONDARY,
-        Error: error
-      })}
-    >
-      <MDEditor
-        value={text}
-        onChange={onChange}
-        commandsFilter={onCommandsFilter}
-        preview={MarkdownEditorModes.EDIT}
-        textareaProps={{
-          placeholder
-        }}
-      />
-      {errorText && <span className={styles.ErrorMessage}>{errorText}</span>}
+    <div className={classNames(styles.Container, className)} data-error={errorText && true}>
+      <div className={styles.Editor}>
+        <MDEditor
+          value={text}
+          onChange={onChange}
+          commandsFilter={onCommandsFilter}
+          preview={MarkdownEditorModes.EDIT}
+          textareaProps={{
+            placeholder
+          }}
+        />
+      </div>
+      {errorText && (
+        <Alert variant={'error'} className={styles.ErrorMessage}>
```

Comment:
I haven't looked at the designs for this MDE - is `error` the only variant this could alert could have? i.e. is there a scenario where we'd show a `warning`?

Reply:
Not sure, this is just how it was written initially. Will add a warning state if we come across it